pkgs/applications/editors/vis/default.nix

```diff
@@ -1,25 +1,30 @@
-{ stdenv, fetchFromGitHub, unzip, pkgconfig, makeWrapper, ncurses, libtermkey, lpeg, lua }:
+{ stdenv, fetchFromGitHub, unzip, pkgconfig, makeWrapper
+, ncurses, libtermkey, lpeg, lua
+, acl ? null, libselinux ? null
+}:
 
 stdenv.mkDerivation rec {
   name = "vis-nightly-${version}";
-  version = "2016-04-15";
+  version = "2016-07-15";
```

Comment:
I've been thinking lately that moving `version`, `sha256`, and `rev` to the attribute set argument on lines 1-5 is a better place for them. If there, they can easily be overridden in a user's "config.nix" file using the `override` command rather than the `overrideDerivation` command, which is a bit more tricky.